src/dashboard/src/fpr/migrations/0029_update_inkscape_svg_command.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-ç
```

Comment:
`ç` -  typo (I think). Not sure it affects anything...

Reply:
Ha! Fixed, thanks.